release-team/role-handbooks/test-infra/README.md

```diff
@@ -9,6 +9,8 @@ There are two major area that test-infra lead need to take care during the relea
 
 1. Create CI/Presubmit jobs for the new release, and populate the testgrid dashboard
 
+1. Configure merge automation for code slush, freeze, and thaw
```

Comment:
nit: this is basically a pseudo table of contents, you could link these to headings below